impit-python/python/impit/impit.pyi

```diff
@@ -0,0 +1,600 @@
+from __future__ import annotations
+
+from typing import Literal
+
+
+Browser = Literal['chrome', 'firefox']
+
+
+class ImpitPyResponse:
+    """Response object returned by impit requests."""
+
+    status_code: int
+    """HTTP status code (e.g., 200, 404)"""
+
+    reason_phrase: str
+    """HTTP reason phrase (e.g., 'OK', 'Not Found')"""
+
+    http_version: str
+    """HTTP version (e.g., 'HTTP/1.1', 'HTTP/2')"""
+
+    headers: dict[str, str]
+    """Response headers as a dictionary"""
+
+    text: str
+    """Response body as text"""
+
+    encoding: str
+    """Response content encoding"""
+
+    is_redirect: bool
+    """Whether the response is a redirect"""
+
+
+class Client:
+    """Synchronous HTTP client with browser impersonation capabilities."""
+
+    def __init__(
+        self,
+        browser: Browser | None = None,
+        http3: bool | None = None,
+        proxy: str | None = None,
+        timeout: float | None = None,
+        verify: bool | None = None,
+    ) -> None:
+        """Initialize a synchronous HTTP client.
+
+        Args:
+            browser: Browser to impersonate ("chrome" or "firefox")
+            http3: Enable HTTP/3 support
+            proxy: Proxy URL to use
+            timeout: Default request timeout in seconds
+            verify: Verify SSL certificates (set to False to ignore TLS errors)
+        """
+
+    def get(
+        self,
+        url: str,
+        content: bytes | bytearray | list[int] | None = None,
+        data: dict[str, str] | None = None,
+        headers: dict[str, str] | None = None,
+        timeout: float | None = None,
+        force_http3: bool | None = None,
+    ) -> ImpitPyResponse:
+        """Make a GET request.
+
+        Args:
+            url: URL to request
+            content: Raw content to send
+            data: unused
```

Comment:
While there are some concerns about the use of `content` / `data` (see #97 ), I wouldn't say `data` is unused (in all the methods). Might be a copy-paste error?